packages/spacebars-compiler/codegen.js

```diff
@@ -27,7 +27,9 @@ var builtInTemplateMacros = {
   // Confusingly, this makes `{{> Template.dynamic}}` an alias
   // for `{{> __dynamic}}`, where "__dynamic" is the template that
   // implements the dynamic template feature.
-  'dynamic': 'Template.__dynamic'
+  'dynamic': 'Template.__dynamic',
+
+  'state': '_.bind(view.templateInstance().state.get, view.templateInstance().state)'
```

Comment:
The resulting JS that gets compiled from templates should look clean. If this dance is what we need, define a utility function in the spacebars package (where the runtime for Spacebars compiled templates live).